airflow/utils/log/bluecore_bq_gcs_task_handler.py

```diff
@@ -0,0 +1,194 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from google.cloud import bigquery
+import os
+from airflow.utils.log.gcs_task_handler import GCSTaskHandler
+from airflow import configuration
+from datetime import datetime
+import subprocess
+import logging
+import string
+try:
+    import ujson as json
+except ImportError:
+    import json
+
+
+
+class BQGCSTaskHandler(GCSTaskHandler):
```

Comment:
This might be a dumb question, but it seems like we changed the KubernetesJobOperator to write logs from the containers as the containers are running.  Doesn't that mean that we no longer need to go to bigquery, and can just retrieve the log from the disk of the airflow worker like other operators do?

Reply:
The reader executes on the `web` node, not the `worker`. Unless we are going to expose the local disk of the worker somehow, we can't read it.